modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
+* A network attachment definition is configured in the same namespace as your VM.
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the VM to which you want to hot plug the network interface is not running, start it by using the following command:
++
+[source,terminal]
+----
+$ virtctl start <vm_name>
+----
++
+.Example VM configuration
```

Comment:
What is this example for?

Reply:
The code block does not quite fit logically. Unless you state otherwise, the assumption is that the code block is the output of the preceding command. 

Do you really need the code block? It just shows a VM connected to the default pod network. Any VM would look like this when it starts.